exe/facter.cc

```diff
@@ -332,9 +332,17 @@ int main(int argc, char **argv)
             auto facts_to_block = vm["blocklist"].as<vector<string>>();
             blocklist.insert(facts_to_block.begin(), facts_to_block.end());
         }
+
+        vector<string> cached_custom_facts;
+        if (vm.count("cached-custom-facts")) {
```

Comment:
we should look if we have a header file where we could add a `const static string` instead hardcoding this value